packages/dds/sequence/src/test/generateSharedStrings.ts

```diff
@@ -86,31 +106,46 @@ export function* generateStrings(): Generator<[string, SharedString]> {
             sharedString.annotateRange(i, i + 10, { bold: true });
         }
 
-        yield [`${version}/withAnnotations`, sharedString];
-
-        sharedString = createNewSharedString();
-        // Very big sharedString
-        for (let i = 0; i < Snapshot.sizeOfFirstChunk; ++i) {
-            sharedString.insertText(0, `${insertText}-${i}`);
-        }
-
-        yield [`${version}/largeBody`, sharedString];
+        yield { snapshotPath: `${version}/withAnnotations`, expected: sharedString, snapshotIsNormalized: normalized };
 
         sharedString = createNewSharedString();
         // SharedString with intervals
         for (let i = 0; i < (Snapshot.sizeOfFirstChunk / insertText.length) / 2; i++) {
             sharedString.insertText(0, `${insertText}${i}`);
         }
 
-        const rand = new Random(Random.engines.mt19937().seed(0));
-        const collection1 = sharedString.getIntervalCollection("collection1");
+        let rand = new Random(Random.engines.mt19937().seed(0));
+        let collection1 = sharedString.getIntervalCollection("collection1");
         collection1.add(1, 5, IntervalType.SlideOnRemove, { intervalId: rand.uuid4() });
 
-        const collection2 = sharedString.getIntervalCollection("collection2");
+        let collection2 = sharedString.getIntervalCollection("collection2");
         for (let i = 0; i < sharedString.getLength() - 5; i += 100) {
             collection2.add(i, i + 5, IntervalType.SlideOnRemove, { intervalId: rand.uuid4() });
         }
 
-        yield [`${version}/withIntervals`, sharedString];
+        yield { snapshotPath: `${version}/withIntervals`, expected: sharedString, snapshotIsNormalized: normalized };
+
+        sharedString = createNewV1SharedString();
+        // SharedString with V1 intervals
+        for (let i = 0; i < (Snapshot.sizeOfFirstChunk / insertText.length) / 2; i++) {
+            sharedString.insertText(0, `${insertText}${i}`);
+        }
+
+        if (version === "v1Intervals") {
+            rand = new Random(Random.engines.mt19937().seed(0));
```

Comment:
rather than copy-paste the code, it's probably worth factoring this out into a util helper that sets up a sharedString you pass it with some intervals.